config/src/main/resources/default-execution-parameters.json

```diff
@@ -5,5 +5,7 @@
   "proto": false,
   "url": null,
   "zipinput": null,
-  "exclude": null
+  "exclude": null,
+  "route__route_sort_order_lower_bound": 0,
+  "route__route_sort_order_upper_bound": 3000
```

Comment:
It's not clear to me what a reasonable upper threshold is for `route_sort_order`, as I haven't seen real data in use for this field. Maybe @timMillet could weigh in?

I don't know if agencies are using some type of scheme for order numbers - for example, first two digits being route number, second two digits being variant of route, like for a route 10:
* 1001
* 1002

...in which case `3000` really won't be a reasonable threshold. 

But if everyone is numbering them 1,2,3 then the current threshold is fine.

Reply:
After discussion with @fabrice-v, it became clear to me how this field is used. If i'm not wrong as you said, fields that represent a sequence order are arbitrary determined by the agencies: they're not numbered the same way from a dataset to another on, which makes it difficult to determine their default values..Hence, I wonder if this PR is relevant at this moment.

Reply:
Correct, it's a sequence number not a key to other data (same as `stop_sequence` and `shape_pt_sequence`. If the code is already implemented you could certainly keep it, although in practice I'm not sure how much it would really be used.